hmpps_person_match/models/person/person.py

```diff
@@ -36,6 +36,13 @@ class Person(BaseModel):
         examples=[["2025-01-01"]],
     )  # Ensures YYYY-MM-DD
 
+    # TOD: Does this need | None?
```

Comment:
There seems to be a slight inconsistency in the DB at the moement where there's unequal treatment of blank lists:
```
select *
from personmatch.person
where sentence_date_first is null
limit 5
```

- `sentence_date_arr`  has `[]` where no sentence dates exist
- `postcode_arr` has `null` where no postcode exists